modules/deploy/partials/high-availability.adoc

```diff
@@ -371,6 +375,160 @@ ID HOST PORT RACK
 
 endif::[]
 
+== Multi-AZ deployments
+
+In a multi-AZ deployment a single Redpanda cluster has brokers distributed over multiple availability zones. With rack awareness, Redpanda places replicas across brokers in different failure zones, resulting in a cluster that can survive a zone outage.
+
+NOTE: Adding a zone does not necessarily increase availability. The replication factor of a given partition is most important. If all of your partitions use a replication factor of three, then adding an additional broker in a fourth zone just means fewer partitions are affected by an outage (since the workload is more spread out).
+
+The primary reason to deploy across multiple availability zones is to achieve extremely high availability, even at the expense of other considerations. Before choosing this approach, carefully consider your system's requirements. Some of the considerations of a multi-AZ approach include:
+
+* Cost: Maintaining presence across multiple availability zones may incur additional costs. You may require additional brokers to hit the minimum requirements for utilizing a multi-AZ deployment. Data sent between availability zones is often chargeable, resulting in additional cloud costs.
+* Performance:  A multi-AZ approach introduces additional message latency. Your brokers are further apart in terms of network distance with additional routing hops in place.
+* Complexity: The Redpanda operational complexity is not appreciably increased, but the complexity of your overall cloud solution is increased. Maintaining presence across availability zones requires additional servers with corresponding maintenance, access control, and standard operational considerations.
+
+=== Multi-AZ infrastructure requirements
+
+Redpanda requires a minimum of three availability zones when using a multi-AZ approach. Deploying across only two availability zones is problematic. For example, given a cluster with three brokers spread across two availability zones, you either end up with all three brokers in one zone or a pair of brokers in one with a single broker in the other. Either way, it's possible to lose a majority of your brokers with a single availability zone outage. You lose the ability to form consensus in affected partitions, negating the high availability state you desire.
+
+=== Multi-AZ optimization
+
+You can configure follower fetching to help ease the cross-AZ cost problems associated with a multi-AZ configuration. This is achieved by configuring consumers to advertise their preferred rack using the `client.rack` option within their consumer configuration. This allows consumers to read data from their closest replica rather than always reading from a (potentially non-local) partition leader.
```

Comment:
Can you please link the first instance of follower fetching here to https://docs.redpanda.com/current/develop/consume-data/follower-fetching/#about-follower-fetching?

Reply:
Done!